### PR TITLE
perf: optimize session file driver

### PR DESCRIPTION
**Pull request by KlassnayaAfrodita**
## 📑 Description
fixed CI
Closes https://github.com/goravel/goravel/issues/?

@coderabbitai summary

## ✅ Checks

- [ ] Added test cases for my code

### CODE REVIEW

File: session/driver/file.go [hunk 1]
```diff
@@ -61,9 +61,10 @@ func (f *File) Open(string, string) error {
 	return nil
 }
 
+//
```
Comment (hwbrzzl) [inline session/driver/file.go:64]: Good catch, thanks, could you remove this line?

Comment (KlassnayaAfrodita) [reply session/driver/file.go]: ye, i remove this line

**Review by devhaozi — APPROVED** [review]
LGTM

**Review by hwbrzzl — APPROVED** [review]
Great, thanks!

**Comment by hwbrzzl** [comment]
Added you to the contributor list, hope to get more contributions from you. :raised_hands:

**Pull request merged**